pmm/pmm2-api-tests.groovy

```diff
@@ -54,10 +54,15 @@ pipeline {
     stages {
         stage('Prepare') {
             steps {
-                // clean up workspace and fetch pmm-api-tests repository
-                deleteDir()
-                git poll: false, branch: GIT_BRANCH, url: 'https://github.com/percona/pmm-managed'
-                slackSend botUser: true, channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+                // fetch pmm-api-tests repository
```

Comment:
Is this comment relevant? We are fetching pmm-managed, not the tests...

Reply:
We just store PMM API tests in pmm-managed now. I'll change commit